src/main/java/seedu/address/logic/commands/FindClientCommand.java

```diff
@@ -0,0 +1,59 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+
+import seedu.address.commons.core.Messages;
+import seedu.address.model.Model;
+import seedu.address.model.appointment.AppointmentContainsKeywordsPredicate;
+import seedu.address.model.property.PropertyClientNamePredicate;
+
+/**
+ * For some keywords, find and list all appointments with names containing those
+ * keywords, and all properties whose clients' names contain them as well.
+ * Keyword matching is case insensitive.
+ */
+public class FindClientCommand extends Command {
+
+    public static final String COMMAND_WORD = "find client";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Finds all properties and appointments involving "
+        + "the person whose name contains the keywords. \ni.e. properties' clients whose names contain the keywords "
```

Comment:
should "the person whose name contains the keywords" from the original address book be removed?

Reply:
I think I meant to say if the user is searching for `bob`, the command will search both lists for properties with client `bob` and appointments with Meet `bob`. I'm using name here since it searches for the string in the `name` attribute of appointments.

Reply:
I see, ok